File: system/monolith/dotnet/Controllers/HealthController.cs
using Microsoft.AspNetCore.Mvc;

namespace MyCompany.MyShop.Monolith.Controllers;

[ApiController]
[Route("")]
public class HealthController : ControllerBase
{
    [HttpGet("health")]
    public IActionResult CheckHealth()
    {
        return Ok(new { status = "UP" });
    }
}

File: system/monolith/java/src/main/java/com/mycompany/myshop/monolith/api/controller/HealthController.java
package com.mycompany.myshop.monolith.api.controller;

import java.util.Map;

import org.springframework.http.ResponseEntity;
import org.springframework.web.bind.annotation.GetMapping;
import org.springframework.web.bind.annotation.RestController;

@RestController
public class HealthController {

    @GetMapping("/health")
    public ResponseEntity<Map<String, String>> checkHealth() {
        return ResponseEntity.ok(Map.of("status", "UP"));
    }
}
